packages/react/src/components/FluidComboBox/FluidComboBox.tsx

```diff
@@ -105,7 +218,7 @@ FluidComboBox.propTypes = {
    * `onChange` is a utility for this controlled component to communicate to a
    * consuming component what kind of internal state changes are occurring.
    */
-  onChange: PropTypes.func,
+  onChange: PropTypes.func.isRequired,
```

Comment:
While this is technically a breaking change, this should've been required from the beginning because it's required in ComboBox and there is no default provided in this component.